### PR TITLE
Feat/change the web link for icons or words

### LINKED ISSUE #1009: Change the Web Link for Icons or Words
Lest's change all this links for words on block producer page 

Image

### PR DESCRIPTION
**Pull request by fvives99**
### What does this PR do?

- Resolve #1009 

### Steps to test

image
image
image
image

- [x] Added Icons to open website and option to copy URL
- [x] Only information that is available is visible (shows email and website as a placeholder until we wait for a Empty State)
image

- [x] Matching style for BP nodes section

### CODE REVIEW

File: webapp/src/components/InformationCard/ProducerInformation.js [hunk 1]
```diff
@@ -1,67 +1,301 @@
...
-            >
-              {value}
-            </Link>
-          ) : (
-            'N/A'
-          )}
-        </Typography>
-      </div>
-    )
+  console.log(info)
```
Comment (AngeloCG97) [inline webapp/src/components/InformationCard/ProducerInformation.js:12]: Please remove this console

File: webapp/src/components/InformationCard/ProducerInformation.js [hunk 2]
```diff
@@ -1,67 +1,301 @@
...
+          {!!info?.ownership ? (
+            <>
+              <>
+                <Typography variant="body1" className={classes.textEllipsis}>
+                  {t('ownershipDisclosure')}
+                </Typography>
+                <LaunchIcon
+                  onClick={() => window.open(info?.ownership, '_blank')}
+                  className={classes.clickableIcon}
+                ></LaunchIcon>
```
Comment (AngeloCG97) [inline webapp/src/components/InformationCard/ProducerInformation.js:146]: Please change 
```
<LaunchIcon
                  onClick={() => window.open(info?.ownership, '_blank')}
                  className={classes.clickableIcon}
                ></LaunchIcon>
```
by 
```
<LaunchIcon
                  onClick={() => window.open(info?.ownership, '_blank')}
                  className={classes.clickableIcon}
                />
```

File: webapp/src/components/InformationCard/ProducerInformation.js [hunk 3]
```diff
@@ -1,67 +1,301 @@
...
+          {!!info?.code_of_conduct ? (
+            <>
+              <>
+                <Typography variant="body1" className={classes.textEllipsis}>
+                  {t('codeofconduct')}
+                </Typography>
+                <LaunchIcon
+                  onClick={() => window.open(info?.code_of_conduct, '_blank')}
+                  className={classes.clickableIcon}
+                ></LaunchIcon>
```
Comment (AngeloCG97) [inline webapp/src/components/InformationCard/ProducerInformation.js:188]: Please change 
```
<LaunchIcon
                  onClick={() => window.open(info?.ownership, '_blank')}
                  className={classes.clickableIcon}
                ></LaunchIcon>
```
by 
```
<LaunchIcon
                  onClick={() => window.open(info?.ownership, '_blank')}
                  className={classes.clickableIcon}
                />
```

File: webapp/src/components/InformationCard/ProducerInformation.js [hunk 4]
```diff
@@ -1,67 +1,301 @@
...
+          {!!info?.chain ? (
+            <>
+              <>
+                <Typography variant="body1" className={classes.textEllipsis}>
+                  {t('chainResources')}
+                </Typography>
+                <LaunchIcon
+                  onClick={() => window.open(info?.chain, '_blank')}
+                  className={classes.clickableIcon}
+                ></LaunchIcon>
```
Comment (AngeloCG97) [inline webapp/src/components/InformationCard/ProducerInformation.js:230]: Please change 
```
<LaunchIcon
                  onClick={() => window.open(info?.ownership, '_blank')}
                  className={classes.clickableIcon}
                ></LaunchIcon>
```
by 
```
<LaunchIcon
                  onClick={() => window.open(info?.ownership, '_blank')}
                  className={classes.clickableIcon}
                />
```

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Review by Edwinjrc — COMMENTED** [review]
LGTM

**Pull request merged**